### PR TITLE
Add new fields to project responses

### PR DESCRIPTION
**Pull request by kleinjm**
### What

Adds mechanism, tagline, state, latitude, longitude, and technology_type to project responses

### Why

Ability to query the new api fields via the sdks

### SDK Release Checklist

- [x] Have you added an integration test for the changes?
- [x] Have you built the gem locally and made queries against it successfully?
- [x] Did you update the changelog?
- [x] Did you bump the package version?
- [ ] For breaking changes, did you plan for the release of the new SDK versions and deploy the API to production?

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -11,5 +11,5 @@
 - [ ] Have you added an integration test for the changes?
 - [ ] Have you built the gem locally and made queries against it successfully?
 - [ ] Did you update the changelog?
-- [ ] Did you bump the package version?
+- [ ] Did you bump the package version [in the code generator](https://github.com/patch-technology/client-code-generation/blob/main/configs/ruby-config.json#L11-L12)?
```
Comment (pcothenet) [inline .github/PULL_REQUEST_TEMPLATE.md:14]: At some point, we might want to clarify guidelines for contributions. E.g. this repo is open-source, but outside users can't really contribute as their changes would get overwritten by the code generator. They also can't access this link.

File: lib/patch_ruby.rb [hunk 2]
```diff
@@ -35,6 +35,7 @@
 require 'patch_ruby/models/order'
 require 'patch_ruby/models/order_list_response'
 require 'patch_ruby/models/order_response'
+require 'patch_ruby/models/parent_technology_type'
```
Comment (pcothenet) [inline lib/patch_ruby.rb:38]: @kleinjm to confirm, this got added automatically, right? (we used to have to do this manually, but hopefully our latest changes fixed this)

**Review by pcothenet — APPROVED** [review]
LGTM!

File: lib/patch_ruby.rb (hunk 2 above)
Comment (kleinjm) [reply lib/patch_ruby.rb:38]: 100% automatic 😎

**Pull request merged**